src/easylink/cli.py

```diff
@@ -37,10 +37,10 @@
         ),
     ),
     click.option(
-        "--timestamp/--no-timestamp",
-        default=True,
-        show_default=True,
-        help="Save the results in a timestamped sub-directory of --output-dir.",
+        "--no-timestamp",
```

Comment:
The autodoc was kinda weird w/ the `--timestamp/--no-timestamp` because it said the default is True (not `--timestamp`). This just seems cleaner to me (and is more in line w/ how we do, e.g. `--pdb`).